src/PrestaShopBundle/Resources/config/services/core/feature_flag.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+
+  PrestaShop\PrestaShop\Core\FeatureFlag\FeatureFlagProviderInterface:
+    alias: PrestaShop\PrestaShop\Core\FeatureFlag\FeatureFlagManager
+    public: true
```

Comment:
Why public true ?

Reply:
Because most of the usages we have so far fetch the service via a `$container->get` We could/should refacto the symfony controllers to inject the service properly but I don't know if our DI is clean enough to do that If it's possible I guess we can try to refacto it in this PR

However the service is also used in legacy controllers so it can only use `$this->get` to access the service, so it needs to be public regardless of the refacto in Symfony

Reply:
I'm not sure this is needed anymore, all controllers are now services, so they can have DI.

Reply:
We actually use this provider to retrieve Feature flag values with `$container->get()` method in some legacy code.
I think in this case, we need to set `public: true` to retrieve the provider, no?